mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java

```diff
@@ -123,12 +123,12 @@ public void showTransactionDetail(Transaction transaction) {
         switch (transaction.getTransactionType()) {
             case DEBIT:
                 isDebit = true;
-                tvOperation.setText("Paid to");
+                tvOperation.setText(R.string.paid_to);
```

Comment:
You have not added the getString here.

Reply:
Sir, it accepts the ageing right away. Didnt find the need for getString()